srml/contract/src/tests.rs

```diff
@@ -73,13 +82,13 @@ type Contract = Module<Test>;
 type System = system::Module<Test>;
 
 pub struct DummyContractAddressFor;
-impl ContractAddressFor<u64> for DummyContractAddressFor {
-	fn contract_address_for(_code: &[u8], _data: &[u8], origin: &u64) -> u64 {
-		origin + 1
+impl ContractAddressFor<H256, u64> for DummyContractAddressFor {
+	fn contract_address_for(code_hash: &H256, data: &[u8], origin: &u64) -> u64 {
```

Comment:
why remove the `_`-prefix if they are still unused?

Reply:
good catch!